include/rocksdb/system_clock.h

```diff
@@ -23,15 +25,19 @@ struct ConfigOptions;
 
 // A SystemClock is an interface used by the rocksdb implementation to access
 // operating system time-related functionality.
-class SystemClock {
+class SystemClock : public Customizable {
  public:
   virtual ~SystemClock() {}
 
   static const char* Type() { return "SystemClock"; }
-
+  static Status CreateFromString(const ConfigOptions& options,
+                                 const std::string& value,
+                                 std::shared_ptr<SystemClock>* result);
   // The name of this system clock
   virtual const char* Name() const = 0;
 
+  static const char* kDefaultName() { return "DefaultClock"; }
```

Comment:
Why we need the kDefaultName() here?

Reply:
There are two classes (Posix and Windows) which are both default, so both of these classes return true for IsInstanceOf(kDefaultName).

Additionally, you can get an instance of the default system clock via SystemClock::CreateFromString(kDefaultName, ...)